_posts/2022-06-28-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+Hello again Trino nation,
+
+We are pleased to announce the 2022 Trino Summit, which will be happening in a
+few short months on November 10th. The event will be *hybrid* this year, meaning
+you can join us online or in-person at the Commonwealth Club right on the
+waterfront in downtown San Francisco, California.
+
+The event will be hosted by Starburst Data, and if you'd like to register to
+attend (online *or* in-person), you can do so on
+[Starburst's registration page.](https://www.starburst.io/info/trinosummit/)
+
+<!--more-->
+
+Perhaps more exciting, if you'd like to submit a paper or give a talk at this
```

Comment:
submit a paper?? this is not a science conference with paper or poster displays..  this is a call for speakers

and maybe also a call for sponsors